examples/webgl_loader_gltf_extensions.html

```diff
@@ -202,14 +202,13 @@
 					var ambient = new THREE.AmbientLight( 0x222222 );
 					scene.add( ambient );
 
-					var directionalLight = new THREE.DirectionalLight( 0xdddddd );
+					var directionalLight = new THREE.DirectionalLight( 0xdddddd, 4 );
```

Comment:
Why the intensity increase here?

Reply:
I think so, yeah. That was a while ago...